.github/workflows/issue-add-support-label.yml

```diff
@@ -0,0 +1,20 @@
+name: Add support label to issue
+
+on:
+  issues:
+    types:
+      - opened
+      - edited
```

Comment:
❌ We should also check for `issue_comments` when `created` or `edited`. We sometimes have a known issue before a support ticket is created. Usually a comment will be placed in the issue linking to the support case.

Reply:
I seems right now this is not supported by the issue labeler action https://github.com/github/issue-labeler/issues/30 :/ 

Generally I would be in favor of always linking the support issue in the issue content, rather than in some comment, to keep all the meta-information on an issue in the main content rather than spread it across comments.
I would adjust the linking instructions on https://confluence.camunda.com/display/HAN/C8+Support+to+Product+Escalation+Process#C8SupporttoProductEscalationProcess-ProductEngineer-CrosslinkJiraandGithubissues accordingly. wdyt?

Reply:
ok, I adjusted the instructions and will also announce/highlight this in the support process slack channel.
If we later on decide this isn't practical we could expand on scanning comments.

Reply:
Sounds good to me 👍